rust/datafusion/src/physical_plan/mod.rs

```diff
@@ -52,6 +52,8 @@ pub type SendableRecordBatchStream = Pin<Box<dyn RecordBatchStream + Send + Sync
 pub enum MetricType {
     /// Simple counter
     Counter,
+    /// Time in nanoseconds
```

Comment:
It would probably help to explicitly mention if this was wall clock time, or cpu time . It looks like this PR saves wallclock time for sort

Both are probably interesting counters / metrics to eventually have